### PR TITLE
Updated project description

### LINKED ISSUE #5: Document problem statement
## Description
As a project reviewer, I want problem statement, scope, user personas, actions, and maybe design to be documented.

## Acceptance Criteria:
- [ ] Document problem statement
- [ ] Document scope/actions
- [ ] Document user personas

### PR DESCRIPTION
**Pull request by haxxorsid**
Updated README.md to:
- [x] Document problem statement
- [x] Document scope/actions
- [x] Document user personas

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,52 @@
 # Referral Board
 A referral network for top tech companies to get referral from verified professionals.
```
Comment (Vineetkhadloya) [inline README.md]: A referral network to get referral from verified professionals.
Looks good !!

**Review by shashank136 — DISMISSED** [review]
great work

**Review by bahetianjali — DISMISSED** [review]

**Review by bahetianjali — APPROVED** [review]
All set!

**Review by shashank136 — APPROVED** [review]
Approved

**Review by Vineetkhadloya — APPROVED** [review]

**Pull request merged**